share/lib/hoc/import3d/import3d_gui.hoc

```diff
@@ -11,6 +11,7 @@ objref file, nil, problist, types, editbox
 objref spine_displ, spine_raw, spine_xyz
 public spine_displ, spine_raw, spine_xyz, sel_spine_ipt, sel_spine_index
 strdef tstr, tstr1, typelabel_, filename
+objref commands
```

Comment:
To allow easy retrieving of commands, shouldn't that variable be public?

Reply:
Ah yes I see, then I don't need the optional parameter in instantiate.
Thank you Michael, I will update the PR

Reply:
Actually, I did not notice that you were indeed returning the commands in $o2. So it's not really necessary to make it public. I gather that part of the usage is to get the commands and then free the import3d_gui.  If this is performance
related, it would be helpful to say that the multiple interpretation of the commands (as opposed to compiling into a
class or function (but that raises issue of size limits) and executing multiple times) does not importantly affect performance.

Reply:
We actually didn't have that in mind, our usage would be to use Import3d_gui to dump the commands as a pre-simulation step (in the sorts of what you were testing out with pickling). By retrieving these commands via import3d_gui we make sure that we don't need to retrofit any changes (like for example the recent format specifier update).

Reply:
Ok. I'm willing to accept the changeset as is with the return in $o2. Or would you rather
retrieve it as public commands.

Reply:
Well since I'm new to hoc, I'd go with whatever you suggest. Though thinking about it, having commands as public looks a bit more elegant to me.